demo/midgard-node/src/workers/utils/common.ts

```diff
@@ -0,0 +1,7 @@
+import { Data } from "effect";
+
+export class WorkerError extends Data.TaggedError("SerializationError")<{
```

Comment:
It's unfortunate that such an error can occur. Can you please make sure no other similar errors are present?
```suggestion
export class WorkerError extends Data.TaggedError("WorkerError")<{
```